### DISCUSSION TITLE
download

### DISCUSSION THREAD

**Discussion by fayaz-bakhsh** [community/community#44303] [Mobile]
hello
add feature about download one file of repository or all of the repository in app!

**Comment by aliazmoodehvaldi** [comment]
By the way, I thought about the same thing a few days ago and I came to the conclusion that this idea is not very practical because tools like downgit.github.io are made for downloading repo.

**Reply by fayaz-bakhsh** [reply]
yes some tools can do this, but should GitHub depend on those tools or be complete by itself? 🤔

**Reply by aliazmoodehvaldi** [reply]
Actually, I agree with you, but there may be other reasons that we are not aware of
